source/_components/input_slider.markdown

```diff
@@ -34,3 +34,71 @@ Configuration variables:
 - **max** (*Optional*): Maximum value for the slider.
 - **step** (*Optional*): Step value for the slider.
 
+# Automation Examples
+
+Here's an example of `input_slider` being used as a trigger in an automation.
+```
+# Example configuration.yaml entry using 'input_slider' as a trigger in an automation
+
+# Define input_slider
+input_slider:
+  bedroom_brightness:
+    name: Brightness
+    initial: 254
+    min: 0
+    max: 254
+    step: 1
+
+# Automation.     
+automation:    
+  - alias: Bedroom Light - Adjust Brightness
+    trigger:
+      platform: state
+      entity_id: input_slider.bedroom_brightness
+    action:
+      - service: light.turn_on
+# Note the use of 'data_template:' below rather than the normal 'data:' if you weren't using an input variable
+        data_template:
+          entity_id: light.bedroom
+          brightness: '{{ trigger.to_state.state | int }}'
```

Comment:
Templates needed to be marked with `{% raw %}...{% endraw %}` otherwise they will get rendered and appear blank.